### PR TITLE
[CI] Add Llama2 to torch compile tests

### PR DESCRIPTION
**Pull request by anko-intel**

### CODE REVIEW

**Review by afierka-intel — APPROVED** [review]
LGTM

File: .jenkins/lm-eval-harness/configs/Llama-2-7B-hf.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# These scores were chosen to place within 6% range of values achieved using  vLLM on HPU:
+# 0.148 - 0.164
+# where on https://www.llama.com/llama2/: 0.146 is given
+model_name: "/mnt/weka/data/pytorch/llama2/Llama-2-7b-hf"
```
Comment (afierka-intel) [inline .jenkins/lm-eval-harness/configs/Llama-2-7B-hf.yaml:4]: Do we really want to you this model? IMO we should use "/mnt/weka/data/pytorch/llama2/Llama-2-7b-chat-hf"

**Review by afierka-intel — CHANGES_REQUESTED** [review]

File: .jenkins/lm-eval-harness/configs/Llama-2-7B-hf.yaml (hunk 1 above)
Comment (anko-intel) [reply .jenkins/lm-eval-harness/configs/Llama-2-7B-hf.yaml:4]: We still have to support and improve LLama2 (as far as I know). I will change the model. Thanks

Comment (anko-intel) [reply .jenkins/lm-eval-harness/configs/Llama-2-7B-hf.yaml:4]: I received bigger than +- 6% spread of results for chat model, so I decided to revert to the original one.

**Pull request merged**